Traceback (most recent call last):
  File "/root/qqq-trading/qqq_trend_tracker.py", line 3, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
Traceback (most recent call last):
  File "/root/qqq-trading/qqq_trend_tracker.py", line 3, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
Traceback (most recent call last):
  File "/root/qqq-trading/qqq_trend_tracker.py", line 3, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
Traceback (most recent call last):
  File "/root/qqq-trading/qqq_trend_tracker.py", line 3, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
Traceback (most recent call last):
  File "/root/qqq-trading/qqq_trend_tracker.py", line 3, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
